pgrx/src/guc.rs

```diff
@@ -55,12 +55,12 @@ pub enum GucContext {
 }
 
 bitflags! {
-    #[derive(Default)]
+    #[derive(Default, Copy, Clone)]
     /// Flags to control special behaviour for the GUC that these are set on. See their
     /// descriptions below for their behaviour.
     pub struct GucFlags: i32 {
         /// Exclude from SHOW ALL
-        const NO_SHOW_ALL = pg_sys::GUC_NO_SHOW_ALL as i32;
+        const NO_SHOW_ALL = pg_sys::GUC_NO_SHOW_ALL as i32 | pg_sys::GUC_NOT_IN_SAMPLE as i32;
```

Comment:
This technically prevents independently setting these. Or at least not in an easy and error-free way.

Not sure I or anyone else cares about that.

Reply:
Just to clarify, you mean it prevents independently setting `pg_sys::NO_SHOW_ALL`, right?

`pg_sys::GUC_NOT_IN_SAMPLE` is not exposed via a safe wrapper as `GucFlags` member, but my understanding is that if/when that is exposed, then that and all the other flags can still be set independently. 

`pg_sys::NO_SHOW_ALL` cannot be set independently but that seems to be a postgres invariant (even though the invariant is not enforced in C)

Reply:
by set independently, I mean if one did this:
```rust
let guc_flags = GucFlags::NO_SHOW_ALL;
let guc_flags = guc_flags & !GucFlags::NOT_IN_SAMPLE;
```
that would be obviously incorrect.
more subtle:
```rust
let guc_flags = GucFlags::NOT_IN_SAMPLE;
let guc_flags = guc_flags & !GucFlags::NO_SHOW_ALL;
```

...I think I'd have to use some function calls on the bitflags API surface instead, since I think it doesn't auto-implement operators, but you get the idea. It does implement a trait that has the equivalent functions as those operators.

Reply:
Anyways, we will worry this slightly unintuitive behavior if we ever have anyone ask for `NOT_IN_SAMPLE`.